blog/meet-content-wro-pazdziernik-2023/index.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Przydałoby się na tym pliku odpalić Prettier. Jeśli masz zainstalowany dodatek Prettier Code Formatter, to po naciśnięciu Ctrl+S (CMD+S na Macu) plik się sformatuje.

Prettier podzieli długie linijki na krótsze, max 80 znaków.